.github/CODEOWNERS

```diff
@@ -150,7 +150,7 @@ sdk/confidentialledger/azure-confidentialledger                      @lynshi
 /sdk/ml/                                                             @paulshealy1 @singankit @diondrapeck @luigiw @harneetvirk @kdestin @MilesHolland @needuv @nthandeMS
 
 # PRLabel: %ML-AutoML
-/sdk/ml/automl.tests.yml                                             @skasturi
+/sdk/ml/automl.tests.yml                                             @skasturi @rtanase @raduk @sharma-riti @jialiu103 @nick863 @anupsms @MaurisLucis @novaturient95
```

Comment:
fyi: looks like jialiu103 and nick863 don't have access to this repo yet

Reply:
Yes, they don't. I am checking with them to raise permission request.